src/App.js

```diff
@@ -1,15 +1,18 @@
+import React from 'react';
 import './App.css';
 import Calculater from './Components/Calculater';
 
-function App() {
-  return (
-    <div className="App">
-      <h1>
-        Hello from App
-        <Calculater />
-      </h1>
-    </div>
-  );
+class App extends React.Component {
+  render() {
+    return (
+      <div className="App">
+        <h1>
+          Hello from App
+          <Calculater />
+        </h1>
+      </div>
+    );
+  }
```

Comment:
- [ ]  I am unable to run your app locally because of an error with the App component. Since the App class component is stateless, I would recommend using `React.PureComponent` instead of `React.Component`.
- The error:
<img width="540" alt="image" src="https://user-images.githubusercontent.com/77190203/170561390-47eee7ef-59e8-4cf1-9450-961c941083e6.png">